dwi_ml/data/dataset/dataset.py

```diff
@@ -336,7 +340,7 @@ def _get_batch_y(self, batch_subj_to_y_id):
         ids = np.arange(len(batch_streamlines))
         self.rng.shuffle(ids)
         flip_ids = ids[:len(ids) // 2]
-        batch_streamlines = [flip_streamline(s) if i in flip_ids else s
+        batch_streamlines = [flip_streamlines(s) if i in flip_ids else s
```

Comment:
`flip_streamlines` is still not implemented, is that intended ?

Reply:
Oups, it will be in PR 20. I just saw a missing s and added it without thinking it was not in the right PR. Sorry. I could remove it but anyway this function is still a big ToDo.

Reply:
Alright no problem !